Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_5_0.md

```diff
@@ -0,0 +1,21 @@
+#### Playbooks
+
+##### Cortex ASM - ASM Alert
+
+- Added an option for Azure in the automated remediation conditional.
+
+##### Cortex ASM - Enrichment
+
+- Added the Cortex ASM - Azure Enrichment playbook as a remediation option.
+
+##### Cortex ASM - Remediation
+
+- Added the Azure Network Security Group Remediation playbook as a remediation option.
+
+##### Cortex ASM - GCP Enrichment
+
+- Update the GCP - Enrichment playbook task to skip this branch if this automation/playbook is unavailable
```

Comment:
Change to:
- Updated the GCP - Enrichment playbook task to skip this branch if this automation/playbook is unavailable.

This should resolve https://github.com/demisto/content/actions/runs/4080101970/jobs/7032208152

Reply:
resolved in https://github.com/demisto/content/pull/24310/commits/abb84a9697d0903de6ff9ebb9928f6e08e3f1e1b